kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java

```diff
@@ -21,10 +21,10 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
You also need to update methods to invert arguments. For example:

this:
https://github.com/fabric8io/kubernetes-client/blob/9c766ffd82248a861845a0d36f2116b91f1221b8/kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java#L278-L279

should be changed to:
```java
 assertTrue(exception.getCause().getMessage().contains("For example java.net.ConnectException"),
      "As the first failure is an IOException the message of the causedBy expected to contain the given text: 'For example java.net.ConnectException'!");
```

Reply:
Yep, makes sense. I have made these changes.